prow/cmd/deck/main.go

```diff
@@ -50,16 +51,41 @@ var (
 	redirectHTTPTo = flag.String("redirect-http-to", "", "host to redirect http->https to based on x-forwarded-proto == http.")
 	// use when behind an oauth proxy
 	hiddenOnly = flag.Bool("hidden-only", false, "Show only hidden jobs. Useful for serving hidden jobs behind an oauth proxy.")
+	runLocal   = flag.Bool("run-local", false, "serve a local copy of the UI, used by the prow/cmd/deck/runlocal script")
```

Comment:
nit: help messages usually start with a capital letter. Same comment about `redirect-http-to`, too.

Reply:
done!